chartpress.py

```diff
@@ -662,6 +696,11 @@ def main(args=None):
         action='store_true',
         help='Print current chartpress version and exit.',
     )
+    argparser.add_argument(
+        '--exit-if-published',
```

Comment:
Do we want this to be the default and opt-out, rather than opt-in? i.e. `--overwrite-chart` and error if already published by default instead?

Reply:
Well I've considered the API that makes sense to implement, and what i concluded isn't reflected in this PR's code yet but only in this comment: https://github.com/jupyterhub/chartpress/pull/75#issuecomment-559023589.

I currently think that overriding charts in a helm chart repo should be considered opt-in, and not be done by default.